src/main/java/com/booking/app/controller/RoomController.java

```diff
@@ -0,0 +1,25 @@
+package com.booking.app.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class RoomController extends BaseController {
+    private static final String API_NAME = "rooms";
+
+    @PostMapping(RoomController.API_NAME + "/{id}")
+    public ResponseEntity<String> createRoom(@PathVariable long id) {
```

Comment:
@RequestBody instead of @PathVariable.
The createRoom method does not {id} in the API.